tests/x11/evolution.pm

```diff
@@ -17,11 +17,13 @@ use testapi;
 
 sub run() {
     x11_start_program("evolution");
+    if ((get_var('FLAVOR') =~ /Gnome-Live/) && check_screen("evolution-preview-release", 20)) {
+        send_key "alt-o";    # OK
+    }
```

Comment:
I like that... thanks... will update evolution.pm - might even consider allowing the 'preview release' in Stagings; as we see it there when we submit Release Candidates (never allowed to checkin to TW though)